bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/editor/EphemerisElementEditor.scala

```diff
@@ -41,22 +43,14 @@ class EphemerisElementEditor(e: EphemerisElement) extends StdModalEditor[Ephemer
     setValue(new Date(e.validAt))
   }
   object RA extends RATextField(e.coords.ra.toAngle.toDegrees)
-  object Dec extends DecTextField(e.coords.dec.toDegrees)
+  object Dec extends DecTextField(e.coords.dec)
```

Comment:
In the previous model both RA and Dec were represented as Doubles (Angle), I just translated this straight to the new model but this breaks on Dec given that it doesn't display correctly for negative angles. I should reconsider this and make `RATextField` operate on `RightAscencion` as you suggest